google-cloud-bigquery/acceptance/bigquery/table_reference_test.rb

```diff
@@ -117,6 +118,30 @@
     partitioned_table.reload!
     partitioned_table.table_id.must_equal partitioned_table_id
     partitioned_table.time_partitioning_type.must_equal "DAY"
+    partitioned_table.time_partitioning_fied.must_be_nil
```

Comment:
NoMethodError: undefined method `time_partitioning_fied' for #<Google::Cloud::Bigquery::Table:0x00007fdcbb5b4e98>
Did you mean?  time_partitioning_field